definition-src/Game/LambdaHack/Definition/Defs.hs

```diff
@@ -1,9 +1,9 @@
 {-# LANGUAGE DeriveGeneric #-}
 -- | Basic types for content definitions.
 module Game.LambdaHack.Definition.Defs
-  ( GroupName, displayGroupName
+  ( GroupName(..), displayGroupName
```

Comment:
Please don't expose the implementation of `GroupName`. If it's hard to just use the exposed functions, let's brainstorm.

Reply:
Did you miss these, because they required "Load more"?

Reply:
I did miss. I can use `displayGroupName`

Reply:
Perfect.